tools/hicup/hicup_hicup.xml

```diff
@@ -0,0 +1,178 @@
+            <data name="dataset2_mapper_barchart" format="svg" from_work_dir="dataset2.mapper_barchart.svg" label="Mapper Dataset2 Barchart.svg" />
+            <!-- Output of the filter step -->
+            <data name="dataset_filt" format="bam" from_work_dir="*.filt.bam" label="filt.bam" />
+            <data name="hicup_filter_summary" format="txt" from_work_dir="hicup_filter_summary*.txt" label="hicup_filter_summary.txt" />
+            <data name="contiguous_filter" format="bam" from_work_dir="hicup_filter_ditag_rejects*/*contiguous.filter.bam" label="contiguous.filter.bam" />
+            <data name="re_ligation_filter" format="bam" from_work_dir="hicup_filter_ditag_rejects*/*re_ligation.filter.bam" label="re_ligation.filter.bam" />
+            <data name="same_dangling_ends_filter" format="bam" from_work_dir="hicup_filter_ditag_rejects*/*same_dangling_ends.filter.bam" label="same_dangling_ends.filter.bam" />
+            <data name="invalid_filter" format="bam" from_work_dir="hicup_filter_ditag_rejects*/*invalid.filter.bam" label="invalid.filter.bam" />
+            <data name="same_circularised_filter" format="bam" from_work_dir="hicup_filter_ditag_rejects*/*same_circularised.filter.bam" label="same_circularised.filter.bam" />
+            <data name="same_internal_filter" format="bam" from_work_dir="hicup_filter_ditag_rejects*/*same_internal.filter.bam" label="same_internal.filter.bam" />
+            <data name="wrong_size_filter" format="bam" from_work_dir="hicup_filter_ditag_rejects*/*wrong_size.filter.bam" label="wrong_size.filter.bam"/>
+            <data name="filter_piechart" format="svg" from_work_dir="*filter_piechart.svg" label="Filter piechart" />
+            <!-- Output of the deduplicator step -->
+            <data name="cis_trans_piechart" format="svg" from_work_dir="*deduplicator_cis_trans_piechart.svg" label="Hicup Deduplicator Cis Trans Piechart.svg"/>
+            <data name="uniques_barchart" format="svg" from_work_dir="*deduplicator_uniques_barchart.svg" label="Hicup Deduplicator Uniques Barchart.svg" />
+            <data name="hicup_deduplicator_summary" format="txt" from_work_dir="hicup_deduplicator_summary*.txt" label="Hicup Deduplicator Summary" />
+        </collection>
+    </outputs>
+    <tests>
+         <test>
+            <!-- inputs -->
+            <param name="input_first_sequence" value="dataset1.fastq" ftype="fastq"/>
+            <param name="input_second_sequence" value="dataset2.fastq" ftype="fastq"/>
+            <param name="re1" value="A^AGCTT"/>
+            <param name="input_files_digest" value="chr1.fa"/>
+            <param name="genome" value="human"/>
+            <conditional name="reference_genome">
+                <param name="source" value="history" />
+                <param name="own_file" value="chr1.fa"/>
+            </conditional>
+
+            <!-- outputs -->
+            <output name="hicup_results" file="results.html" ftype="html" lines_diff="10000"/>
```

Comment:
If we really want to keep this tool, we should also keep the test file. This was removed in this PR.
Or do we want to remove this tool altogether?

Reply:
Sorry I thought I put it. I will put it back.
I think it is better to keep this tool.

Reply:
The result.html is in my github. Do I need to do a new pull request or how can I do?

Reply:
No, perfect. If you look in a little bit down you see your new commits and the added file.